apps/web/app/gallery/[slug]/page.tsx

```
@@ -108,104 +109,106 @@ export default async function GalleryGroupPage({ params }: PageProps) {
-                        <h1 className='mb-6 font-serif text-4xl text-stone-900 md:text-5xl lg:text-6xl'>
-                            {group.name}
-                        </h1>
-
-                        {group.description && (
-                            <p className='text-lg leading-relaxed font-light text-stone-600 md:text-xl'>
-                                {group.description}
+            <ContainerLayout as='div' noPadding size='full'>
+                {/* ImageGallery Schema */}
+                <script
+                    type='application/ld+json'
+                    dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)